acceptance/tests/install/from_file.rb

```diff
@@ -0,0 +1,18 @@
+test_name 'test generic installers'
+
```

Comment:
@johnduarte I was curious about the case when the operating system was neither `osx` or `win`; in this case, the test will indeed pass, but nothing will have actually really been tested. Should this have a `skip_test` condition at the top?